app/frontend/src/utils/constants.js

```diff
@@ -66,10 +66,32 @@ export const IdentityProviders = Object.freeze({
   IDIR: 'idir' // IDIR
 });
 
-/** Corresponds to vuetfy alert classes for notification types */
+/** Corresponds to vuetify alert classes for notification types */
 export const NotificationTypes = Object.freeze({
-  ERROR: 'error',
-  SUCCESS: 'success'
+  ERROR: {
+    type: 'error',
+    class: 'alert-error',
+    color: '#f2dede',
+    icon: 'error',
+  },
+  SUCCESS: {
+    type: 'success',
+    class: 'alert-success',
+    color: '#dff0d8',
+    icon: 'check_circle',
+  },
+  INFO: {
+    type: 'info',
+    class: 'alert-info',
+    color: '#d9eaf7',
+    icon: 'info',
+  },
+  WARNING: {
+    type: 'warning',
+    class:'alert-warning',
+    color: '#f9f1c6',
+    icon: 'warning',
+  },
```

Comment:
I don't _think_ we want to define things like colors etc in code rather than style if avoidable.
I think we just want the invoker designating the type of alert and then stylesheets determining how those alert types look. Let's discuss with the team.

Reply:
Yea - we definitely do NOT want to be defining color constants in the javascript portion of our code - this needs to be defined in our SASS area.

Reply:
Will be removing these styling calls from my next commit.